control/control.openSUSE.xml

```diff
@@ -185,6 +185,15 @@ textdomain="control"
                     <install_patterns>lxde</install_patterns>
                 </window_manager>
             </window_managers>
+            <!-- Do not ask for vendor change during this defined upgrade -->
+            <product_upgrade>
```

Comment:
We should allow multiple definitions here (https://github.com/yast/yast-installation-control/pull/102#pullrequestreview-504497310) and then we could add here two definitions. One for "openSUSE Jump 15.2.1" and for "openSUSE Jump".

So it would work with the current Jump and also after fixing https://bugzilla.suse.com/show_bug.cgi?id=1177443.